document/FileFormat.mdk

```diff
@@ -2696,31 +2708,385 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | `annotatedCodeLocation`     |                             |                             |
+|                  | object shall be created     |                             |                             |
+|                  | for each alias that the     |                             |                             |
+|                  | tool creating the log       |                             |                             |
+|                  | wishes to represent.        |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `assignment`     | Recommended: The            | Recommended: The            | Optional: the names and     |
+|                  | name of the variable        | value assigned to           | values of the variables     |
+|                  | being assigned to.          | the target variable.        | participating in the        |
+|                  |                             |                             | expression on the right-    |
+|                  |                             |                             | hand side of the            |
+|                  |                             |                             | assignment.                 |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `branch`         | Recommended: If the         | Optional: If the branch     | Optional: If the branch is  |
+|                  | target of the branch is     | is the result of a test,    | the result of a test,       |
+|                  | a named label, in           | in which case, the          | in which case, the names    |
+|                  | which case, the name        | Boolean value of the test   | and values of the variables |
+|                  | of the label.               | condition.                  | participating in the        |
+|                  | Optional: If the target     | Absent: If the branch is    | expression being tested.    |
+|                  | of the branch is a          | unconditional.              | Absent: If the branch is    |
+|                  | known address or            |                             | unconditional.              |
+|                  | offset, in which case,      |                             |                             |
+|                  | a hexadecimal               |                             |                             |
+|                  | string representation       |                             |                             |
+|                  | of that address or          |                             |                             |
+|                  | offset.                     |                             |                             |
```

Comment:
It we provide targetLocation, then this optional text goes away. Where'd we land on that issue?